src/layouts/docs/index.tsx

```diff
@@ -39,6 +41,10 @@ const DocsLayout: React.FC<DocsLayoutProps> = (props) => (
                 label="Edit this page on GitHub"
               />
             )}
+            <Footer
+              className={s.footer}
+              openConsentManager={props.openConsentManager}
+            />
```

Comment:
Did you already try showing the footer by passing props to the `BaseNewLayout` this layout renders above?

Reply:
Yep - it ends up rendering the footer below the content area, rather than within it as spec'd in the designs:

<img width="1440" alt="CleanShot 2022-01-12 at 18 17 18@2x" src="https://user-images.githubusercontent.com/4624598/149238812-975d960e-04c6-4f3f-8c0b-e686685ef29f.png">

Have asked design about this since I think it would be nice to take a more consistent approach to footer (#77 implements a way for us to easily switch between em, if needed). I think there's a decent possibility that we may want a more robust, site-mappy footer like the one we have on [hashicorp.com](https://www.hashicorp.com), in which case I think we'll something closer to the approach you're suggesting 👍

Reply:
Ah, I see why you took your approach now! I didn't fully notice that the footer is moved over and next to the sidebar on these pages. My take right now (and out of scope of this PR since there'd need to be a wide discussion) is that the footer should take up the full width for the same reasons main navigation headers take up the full width, but what you've got here is great and to-spec. ✨